spec/observables/fromEvent-spec.js

```diff
@@ -127,4 +127,81 @@ describe('Observable.fromEvent', function () {
 
     send('test');
   });
-});
\ No newline at end of file
+
+  it('should not fail if no event arguments are passed and the selector does not return', function (done) {
+    var send;
+    var obj = {
+      on: function (name, handler) {
+        send = handler;
+      },
+      off: function () {
+      }
+    };
+
+    function selector() {
+    }
+
+    Observable.fromEvent(obj, 'click', selector).take(1)
+      .subscribe(function (e) {
+        expect(e).toBeUndefined();
+      }, function (err) {
+        done.fail('should not be called');
+      }, done);
+
+    send();
+  });
+
+  it('should return a value from the selector if no event arguments are passed', function (done) {
+    var send;
+    var obj = {
+      on: function (name, handler) {
+        send = handler;
+      },
+      off: function () {
+      }
+    };
+
+    function selector() {
+      return 'no arguments';
+    }
+
+    Observable.fromEvent(obj, 'click', selector).take(1)
+      .subscribe(function (e) {
+        expect(e).toBe('no arguments');
+      }, function (err) {
+        done.fail('should not be called');
+      }, done);
+
+    send();
+  });
+
+  it('should pass multiple arguments to selector from event emitter', function (done) {
+    var send;
+    var obj = {
+      on: function (name, handler) {
+        send = handler;
+      },
+      off: function () {
+      }
+    };
+
+    function selector(x, y, z) {
+      return {
+        arg1: x,
+        arg2: y,
+        arg3: z
+      };
```

Comment:
I was thinking more like:  `return [].slice.call(arguments)` here, and a check of `expect(e).toEqual([1,2,3])` below because what's currently here won't catch a bug where additional arguments are unintentionally passed. 

Nitpicky, I know. :)

I really appreciate your effort on this PR.